packages/headless/src/controllers/commerce/context/cart/headless-cart.ts

```diff
@@ -35,41 +60,65 @@ export interface CartProps {
  */
 export interface Cart extends Controller {
   /**
-   * Replaces the cart items.
-   * @param items - The new cart items.
-   */
-  setItems(items: CartItem[]): void;
-
-  /**
-   * Adds a cart item.
-   * @param item - The new cart item.
+   * Creates, updates, or deletes an item in the cart, and emits an `ec.cartAction` analytics event if the `quantity` of
```

Comment:
I remember seeing an RFC on naming events with a `ec.` vs `ec_` prefix. Has this been settled?

Reply:
Not that I'm aware of, but we'll update the doc if needed 😄